node_cli/realm/realmTasks.js

```diff
@@ -0,0 +1,101 @@
+var Realm = require("realm");
+var mongodb = require("mongodb");
+const schemas = require("./schemas");
+const index = require("../index");
+
+async function setRealm(user) {
+  const config = {
+    schema: [schemas.TaskSchema, schemas.UserSchema, schemas.ProjectSchema],
+    sync: {
+      user: user,
+      partitionValue: "myPartition",
+    },
+  };
+
+  return await Realm.open(config).then((r) => {
+    return r;
+  });
```

Comment:
The `.then()` here is redundant. You can just `return await Realm.open(config);` (or even just `return Realm.open(config)`)